test/controllers/forms/SelectedSchemesFormSpec.scala

```diff
@@ -25,25 +25,14 @@ class SelectedSchemesFormSpec extends PlaySpec {
   "Selected Schemes form" should {
     "be valid when required values are supplied" in {
        val form = selectedSchemesForm.bind(Map("scheme_0" -> "Finance", "orderAgreed" -> "true",
-         "eligible" -> "true", "alternatives" -> "false"))
+         "eligible" -> "true"))
        form.hasErrors mustBe false
        form.hasGlobalErrors mustBe false
     }
 
     "be invalid when schemes are not supplied" in {
       val form = selectedSchemesForm.bind(Map("orderAgreed" -> "true",
-        "eligible" -> "true", "alternatives" -> "true"))
-      form.hasErrors mustBe true
-      form.hasGlobalErrors mustBe false
-    }
-
-    "be invalid when more than 5 schemes are supplied" in {
```

Comment:
Instead of removing this test shall we modify this test to cover the scenario of multiple scheme selection.